MediaBrowser.Model/Configuration/EncodingOptions.cs

```diff
@@ -36,7 +36,8 @@ public class EncodingOptions
         public string EncoderPreset { get; set; }
 
         public string DeinterlaceMethod { get; set; }
-        public bool EnableDecodingColorDepth10 { get; set; }
+        public bool EnableDecodingColorDepth10Hevc { get; set; }
+        public bool EnableDecodingColorDepth10Vp9 { get; set; }
```

Comment:
To be more future-proof we might want to nest a structure of bools, so it would become `EnableDecodingColorDepth10.CodecName`. Could be done later if needed, though.